lib/ansible/modules/cloud/vmware/vcenter_license.py

```diff
@@ -95,9 +116,8 @@
 
 try:
     from pyVmomi import vim
-    HAS_PYVMOMI = True
 except ImportError:
-    HAS_PYVMOMI = False
+    pass
```

Comment:
We actually use `vim` below. If you want to simplify the code, I would prefer that you completely remove the try/except block instead and let Python raises the exception.

Reply:
Pyvmomi class handles `HAS_PYVMOMI` exceptions so this is not required here.

Reply:
Oh. indeed.